test/remote/usps_test.rb

```diff
@@ -125,6 +125,70 @@ def test_international_rates
     assert_equal [], other_than_two, "Some RateEstimates do not refer to the right number of packages (#{other_than_two.inspect})"
   end
 
+  def test_domestic_label_from_name_errros
+    assert_raises(ResponseError, "From Name is required and may not exceed 100 characters.") do
+      @carrier.create_shipment(
+        location_fixtures[:new_york],
+        location_fixtures[:beverly_hills_with_name],
+        package_fixtures.values_at(:book),
+        :test => true,
+        :service => :first_class
+      )
+    end
+  end
+
+  def test_domestic_label_to_name_errors
+    assert_raises(ResponseError, "To Name is required and may not exceed 100 characters.") do
```

Comment:
`assert_raises` doesn't actually verify that the string you pass in is the exception message - it's actually the assertion failure message. You can see it being `pop`'d from `*exp` here: `https://github.com/seattlerb/minitest/blob/master/lib/minitest/assertions.rb#L319`

It's rather misleading, so to verify an exception message what you want to do is actually set the return value of `assert_raises` to a variable, which will hold the exception message. From there, you can call something like `assert_equal "your exception message", exception.message`.